quickwit-cli/src/index.rs

```diff
@@ -240,9 +256,44 @@ impl IndexCliCommand {
         }))
     }
 
-    fn parse_demux_args(_matches: &ArgMatches) -> anyhow::Result<Self> {
-        // TODO
-        Ok(Self::Demux(DemuxArgs {}))
+    fn parse_merge_args(matches: &ArgMatches) -> anyhow::Result<Self> {
+        let metastore_uri = matches
```

Comment:
nit: I introduced the bad pattern in the first place... but I believe `expect` is better than `context` because we own and configure the CLI args:
```rust
        let metastore_uri = matches
            .value_of("metastore-uri")
            .map(normalize_uri)
            .expect("`metastore-uri` should be a required arg.")?;
```

Reply:
let's do some search & replace

Reply:
Do we want to handle this later?

Reply:
The YAML file ensures that we have all the required arguments here.
Arg for argument is ok I think.